api/tests/opentrons/protocol_api/test_parameter_context.py

```diff
@@ -9,6 +9,7 @@
     MAX_SUPPORTED_VERSION,
 )
 from opentrons.protocols.parameters import (
+    csv_parameter_definition as mock_cvs_parameter_definition,
```

Comment:
```suggestion
    csv_parameter_definition as mock_csv_parameter_definition,
```
..unless you want to call it CVS definition :P

Reply:
🤦